topo2geo/core.py

```diff
@@ -34,17 +34,24 @@ def main(input_file, output_file):
         print(f'Error: Input file {input_file} does not exist.')
         return
 
-    try:
-        geojson = to_geojson(input_file)
-        with open(output_file, 'w+') as dest:
-            dest.write(json.dumps(geojson))
-    except AssertionError:
-        print('Error: Invalid TopoJSON')
-    except ValueError as e:
-        print('Error: Invalid TopoJSON')
-        print(f'{e}')
-    except IOError:
-        print('Error: Issue reading file')
+    geojson_layers = to_geojson(input_file)
+
+    for layer, geojson in geojson_layers.items():
+        if len(geojson_layers.keys()) > 1:
+            geojson_fn = f'{layer}_{output_file}'
```

Comment:
I think it would be better formatting to put the output filename first and the layer after, that way if someone making a lot of layers they can be sorted correctly in their filesystem

Reply:
I agree. It's not complicated to make that happen. I've added a new commit with the changes.